### DISCUSSION TITLE
$$Remove_fixities  is not updated in ANURA3D_2023

### DISCUSSION THREAD

**Discussion by EmreDumlu-UM** [Anura3D/Anura3D_OpenSource#154] [General]
Dear All,
After activating the removing fixities in GID, the section of $$Remove_fixities was not updated in the generated .CPS file. Please see the related figure below. I change them manually to 1, 1, 0 for solid and liquid fixities. However, especially in single-point (liquid) cases, I have to remove both solid and liquid fixities even if I am dealing with only “liquid” simulations. This a little confuses me. I define liquid which is the only material type in my domain, however, I have to remove the solid fixities as well. 

![removing_fixities](https://github.com/Anura3D/Anura3D_OpenSource/assets/125078884/4fe6aa07-ec00-4382-bf75-d3c1bad118aa)

**Comment by pmarveggio** [comment]
@gdicarluccio updated the ProblemType to fix the gom generation bug. I will investigate the liquid only simulation bug and let you know
